src/vivarium_gates_mncnh/components/intrapartum.py

```diff
@@ -74,9 +74,14 @@ def on_time_step(self, event: Event) -> None:
             coverage_value,
         ) in self.coverage_values.items():
             facility_idx = pop.index[pop[COLUMNS.DELIVERY_FACILITY_TYPE] == facility_type]
+            coverage_vlaue = (
```

Comment:
This had been in another PR (since the model would not of worked without this) so Im not sure why it was not in this branch.